bpfman/src/cli/args.rs

```diff
@@ -6,7 +6,11 @@ use clap::{Args, Parser, Subcommand};
 use hex::FromHex;
 
 #[derive(Parser, Debug)]
-#[command(author, version, about, long_about = None)]
+#[command(
+    long_about = "CLI for interacting with bpfman. Used to load, unload and view eBPF programs on a system."
```

Comment:
This should say something like `An eBPF Program Manager built for Linux and Kubernetes` or whatever we use in our README and or Cargo.toml as our description.

Reply:
Done